package-lock.json

```diff
@@ -0,0 +1,15738 @@
+{
```

Comment:
Do we want to check in `package-lock.json`? I don't think it is being done in the main branch. It's pretty annoying but I also get that its useful for consistency. If I were to vote on it, I'd vote to not check it in.

Reply:
ok. my reason for package-lock is it is a really nice feature in Node for CI/CD, since it creates reproducible environments for testing. It adds some "bloat" to PRs but the gain of consistent dependencies for testing is worth it IMHO.